src/cursor/find_cursor.ts

```diff
@@ -49,6 +49,16 @@ export class FindCursor extends AbstractCursor {
     }
   }
 
+  /**
+   * Returns a new unitialized copy of this cursor, with options matching those that have been set on the current instance
```

Comment:
```suggestion
   * Returns a new uninitialized copy of this cursor, with options matching those that have been set on the current instance
```

Reply:
fixed here and in aggregation_cursor too!